cookiecutter.json

```diff
@@ -1,11 +1,20 @@
 {
-	"author_name": "Elixir Cloud AAI",
-	"author_email": "cloud-service@elixir-europe.org",
-	"development_status": "Alpha",
-	"short_description": "A short description of the project.",
-	"project_slug": "python_cookiecutter",
-	"github_username": "elixir-cloud-aai",
-	"python_version": "3.11",
-  "add_script": "y",
-  "year": "2024"
+    "author_name": "ELIXIR Cloud AAI",
+    "author_email": "cloud-service@elixir-europe.org",
+    "development_status": [
+        "1 - Planning",
+        "2 - Pre-Alpha",
+        "3 - Alpha",
+        "4 - Beta",
+        "5 - Production/Stable",
+        "6 - Mature",
+        "7 - Inactive"
+    ],
+    "short_description": "A short description of the project.",
+    "project_name": "python-cookiecutter",
+    "project_slug": "{{ cookiecutter.project_name.lower().replace('-', '_') }}",
+    "github_username": "elixir-cloud-aai",
+    "python_version": "3.11",
+    "add_script": "y",
```

Comment:
hmmm.... my reasoning was the y, will alwasy create a main file, if alot of proj end up using this template, everyone of the proj will have `main.py` as the entry point of the app, instead of dev inventing their own lingo. What do you think?

Reply:
Makes sense